spm-client-common-libs-parent/pom.xml

```diff
@@ -79,7 +79,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.12.7.1</version>
+      <version>10.0.pr1</version>
```

Comment:
This version is a bit weird, maybe a typo?

Reply:
right it's a typo, the version is 2.10.0.pr1 thanks